sandbox/pages/SandboxCard.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="kcard-sandbox">
```

Comment:
nit: this class is confusing since it's similar to k-card

Reply:
Literally every component sandbox page has a wrapping class of `.k[component]-sandbox`. I don't think it's confusing since it has a word sandbox in it.

Reply:
kk resolving 👍🏼